src/inmanta/server/services/orchestrationservice.py

```diff
@@ -58,7 +58,7 @@
 from inmanta.server.agentmanager import AgentManager, AutostartedAgentManager
 from inmanta.server.services.resourceservice import ResourceService
 from inmanta.server.validate_filter import InvalidFilter
-from inmanta.types import Apireturn, JsonType, PrimitiveTypes
+from inmanta.types import Apireturn, JsonType, PrimitiveTypes, ReturnTupple
```

Comment:
Oh, there's a typo in this one... First time I notice. Little to be done about it though.

Reply:
damn, and this time the author is no who you suspect first ;-)